static/src/javascripts/projects/common/views/commercial/survey/survey-simple.html

```diff
@@ -22,6 +22,16 @@ <h3 class="survey-text__header">
                 <p class="survey-text__info">
```

Comment:
Maybe instead of using separate variables - `surveyTextFirst`, `surveyTextSecond`, etc - you could just put the paragraphs in an array, e.g.

```html
<% surveyText.forEach(function(para) {%>
    <p class='survey-text__info'>
        <%=para%>
    </p>
<% }) %>
```